plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcWriteConfig.java

```diff
@@ -57,4 +59,19 @@ public JdbcWriteConfig setNonTransactionalInsert(boolean nonTransactionalInsert)
         this.nonTransactionalInsert = nonTransactionalInsert;
         return this;
     }
+
+    @Min(1)
+    @Max(128)
```

Comment:
Isn't the max also capped by number of nodes * task writer count per node? Can you test what happens when set to max?